eng/tools/rush-runner/src/rush.js

```diff
@@ -52,6 +53,32 @@ export function rushRunAllWithDirection(action, packagesWithDirection, rushParam
     invocation,
   });
 
+  // Restore assets for packages that are being 'unit-test'-ed in the CI pipeline
+  if (
+    // 1. Check if running in Public CI (process.env["BUILD_BUILDNUMBER"] is set)
+    process.env["BUILD_BUILDNUMBER"]
```

Comment:
So this will _work_, but I think you should check two other variables.

- `TF_BUILD` not empty / true = in azure devops
- `CI` not empty / true = in github actions

Reply:
Or maybe a `--ci` flag or something that the CI passes in could be used to distinguish instead of checking these environment variables? Then we wouldn't be in the business of detecting whether we're in CI anymore here, feels like it would be less coupled that way. I don't feel strongly though if doing it that way is more difficult

Reply:
```suggestion
    process.env["BUILD_BUILDNUMBER"] && process.env["TF_BUILD"] && process.env["CI"]
```
is this it?

Reply:
I agree with the top level CI flag. Sometimes it's useful to run in CI mode locally for debugging purposes, and the fewer implicit environment variables there are, the easier this is to do.